xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/displayer_page.vm

```diff
@@ -1 +1,40 @@
-#template('displayer_dblist.vm')
+#if ($type == 'edit' || $type == 'search')
+  #if ($field.getProperty('displayType').value == 'input' && $field.getProperty('picker').value == 1)
+    #xpropertySuggestInputDisplayer($field $prefix $name $value)
+  #else
+    $doc.displayEdit($field, $prefix, $object)
+  #end
+#elseif ($type == 'view')
+  #set ($pages = [])
+  #foreach($pageReference in $value)
+    #set ($discard = $pages.add("#displayPageReference($pageReference)"))
+  #end
+  #if ($pages.size() > 0)
+    $stringtool.join($pages, '-')
```

Comment:
I'm not sure about this separator, we don't use it elsewhere. Why not use a comma ``,``?

Reply:
@mflorea Pages ended up separated with a comma and spaces like that: `Page1 , Page2 , Page3`. I'll fix the issue.